spec/queries/vms_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "Vm queries" do
```

Comment:
Maybe break this down with another describe below actually specifying the field you're testing:

```
RSpec.describe "Vm queries" do
  describe "'vm' field" do
  end

  describe "'vms' field" do
  end
end
```

Maybe?